lib/fluent/plugin/in_systemd.rb

```diff
@@ -76,12 +75,9 @@ def formatted(entry)
       end
 
       def watch
-        while @running
-          next unless @journal.wait(1_000_000)
```

Comment:
We should check that loosing this line does not effect performance, journal.wait returns straightaway if there are new events on the journal, here I think we are going to wait a full second once we have reached the end...